.travis.yml

```diff
@@ -35,10 +35,10 @@ script:
 after_success:
   - scripts/deploy.sh netlify
   - scripts/deploy-elasticbeanstalk.sh
-  - scripts/push-image.sh unlock
-  - scripts/push-image.sh unlock-integration
+  - test $TRAVIS_BRANCH = "master" && test $TRAVIS_PULL_REQUEST = "false" && scripts/push-image.sh unlock $TRAVIS_BRANCH
```

Comment:
Note that instead of pushing to `latest` we will be pushing to `master`.

Reply:
Are these setting the environment variables or checking for equality?

Reply:
Checking for equality ;)